hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/KeepAliveCache.java

```diff
@@ -96,6 +97,8 @@ class KeepAliveCache extends Stack<KeepAliveCache.KeepAliveEntry>
    */
   private final AtomicBoolean isPaused = new AtomicBoolean(false);
 
+  private final String accountNamePath;
```

Comment:
add javadoc to say this is only used for exception messages.

Reply:
Make sense. Have taken it.